src/pages/cli/geo/geofencing.mdx

```diff
@@ -0,0 +1,289 @@
+export const meta = {
+  title: `Geofencing (Developer Preview)`,
+  description: `Use Amplify CLI to create and manage collections of Geofences`,
+};
+
+Amplify's `geo` category enables you to create and manage collections of Geofences used to setup virtual geographic perimeters.
+
+<Callout>
+
+**Note:** The Amplify Geo Geofencing feature is in developer preview and is not intended to be used in production environments. Please reach out to us for any feedback and/or issues [here](https://github.com/aws-amplify/amplify-cli/issues)
+
+</Callout>
+
+## Install Amplify CLI
+Since the Geofencing feature is a developer preview, you need to update your Amplify CLI to try it out:
+
+```bash
+npm i -g @aws-amplify/cli@geo
+```
+
+<Callout>
+
+**Note:** The version of Amplify CLI should be `7.3.2-geo.3` or later. You can check the version using `amplify -v`.
+
+</Callout>
+
+## Setup a new Geofence Collection
+Before you add a new Geofence Collection to your application, you need to have at least one Cognito User Pool group added to your project.
```

Comment:
Is it intended that we need a Cognito User Pool _group_? Or just a Cognito User Pool?